test/fixtures/cmd/podman-errors

```diff
@@ -1,2 +1,6 @@
 Error: inspecting object: network min: network not found
-Error: inspecting object: unable to inspect \"abc\": failed to find image abc: abc: image not known
\ No newline at end of file
+Error: inspecting object: unable to inspect \"abc\": failed to find image abc: abc: image not known
+Error: no pod with name or ID non_existing_pod found: no such pod
+[]
+error inspecting object: no such object: "non_existing_volume"
+Error: inspecting object: no such object: "volume"
```

Comment:
better to split this into different fixtures

Reply:
This is fixtures and can be treated as list of errors. As this is unit testing it's mot going to make differnce as we are not doing any functional testing.